### ISSUE TITLE
Web viewer issue

### ISSUE THREAD

**Issue by ZeWang95** [traveller59/second.pytorch#89]
Hi,

When I'm trying the viewer, after I enter the frontend url, I got something like this:
![image](https://user-images.githubusercontent.com/41649297/52695496-b7d04880-2f3a-11e9-80d7-f62d5efe49b3.png)
I don't know how I'm supposed to continue the following step.
Any idea what the issue might be?

Thanks a lot!

**Comment by MadhavEsDios** [comment]
Duplicate of my issue #86, just remove the integrity field in the index.html file

**Comment by lawrencekiba** [comment]
Same issue, do you need to delete all the integrity fields to make it work @MadhavEsDios ?

**Comment by MadhavEsDios** [comment]
Yes @lawrencekiba , deleting the content of integrity field and turning them blank did the job for me.
